src/components/PostDetails/PostDetails.jsx

```diff
@@ -1,45 +1,77 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+
 import { NewCommentForm } from '../NewCommentForm';
 import './PostDetails.scss';
+import { getPostComments, remove } from '../../api/comments';
 
-export const PostDetails = () => (
-  <div className="PostDetails">
-    <h2>Post details:</h2>
+export const PostDetails = ({ postId, body }) => {
+  const [postComments, setPostComments] = useState('');
+  const [isHide, setIsHide] = useState(false);
 
-    <section className="PostDetails__post">
-      <p>sunt aut facere repellat provident occaecati excepturi optio</p>
-    </section>
+  useEffect(() => {
+    getPostComments(postId)
+      .then(result => setPostComments(result));
+  }, [postId, postComments]);
```

Comment:
ти встановлюєш нові коментарі від чого міняються коментарі від чого виклакається функція в useEffect яка міняє коментарі що викликає useEffect.......    Розумієш в чому проблема, не шукай легких шляхів, придумай інший спосіб щоб після видалення і створення коментаря вони відразу там зявлялись
а тут видали postComments з масиву залежностей
```suggestion
  }, [postId]);
```